manifests/s/SAGAUserGroupAssociation/SAGAGIS/9.1.1/SAGAUserGroupAssociation.SAGAGIS.installer.yaml

```diff
@@ -0,0 +1,33 @@
+# Created with YamlCreate.ps1  v2.2.9 $debug=QUSU.CRLF.5-1-22621-1778.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: SAGAUserGroupAssociation.SAGAGIS
+PackageVersion: 9.1.1
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: machine
+InstallerSwitches:
+    Silent: "/VERYSILENT"
+InstallModes:
+- silent
+UpgradeBehavior: install
+FileExtensions:
+- sg-grd-z
+- sg-pts
+- sg-pts-z
+- sgrd
+- shp
+- spc
+- sprj
+Installers:
+- Architecture: x64
+  InstallerUrl: https://deac-ams.dl.sourceforge.net/project/saga-gis/SAGA%20-%209/SAGA%20-%209.1.1/saga-9.1.1_x64_setup.exe
+  InstallerSha256: CBFD6D991DF3B7C59C9934178D64D5FB0C42B2ACF25713C33A7CABB7C002958F
+  ProductCode: SAGA - System for Automated Geoscientific Analyses_is1
+- Architecture: x86
+  InstallerUrl: https://deac-riga.dl.sourceforge.net/project/saga-gis/SAGA%20-%209/SAGA%20-%209.1.1/saga-9.1.1_x64_setup.exe
+  InstallerSha256: CBFD6D991DF3B7C59C9934178D64D5FB0C42B2ACF25713C33A7CABB7C002958F
+  ProductCode: SAGA - System for Automated Geoscientific Analyses_is1
```

Comment:
```suggestion
```

Reply:
Hi @xyzallan,

This appears to be a 64-bit only installer. Is that accurate?

Reply:
It seems so yes.